File: app/src/main/java/com/lifedawn/bestweather/retrofit/responses/accuweather/Wind.java
package com.lifedawn.bestweather.retrofit.responses.accuweather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Wind {
	@Expose
	@SerializedName("Speed")
	private ValuesUnit speed;
	
	@Expose
	@SerializedName("Direction")
	private Direction direction;
	
	public ValuesUnit getSpeed() {
		return speed;
	}
	
	public Wind setSpeed(ValuesUnit speed) {
		this.speed = speed;
		return this;
	}
	
	public Direction getDirection() {
		return direction;
	}
	
	public void setDirection(Direction direction) {
		this.direction = direction;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/accuweather/Wind.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.accuweather

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Wind {
    @Expose @SerializedName("Speed") var speed: ValuesUnit? = null
        private set
    @Expose @SerializedName("Direction") var direction: Direction? = null
    fun setSpeed(speed: ValuesUnit?): Wind {
        this.speed = speed
        return this
    }
}